WalletWasabi/Blockchain/Transactions/TransactionFactory.cs

```diff
@@ -232,41 +231,21 @@ public TransactionFactory(Network network, KeyManager keyManager, ICoinsView coi
 				builder = builder.AddKeys(signingKeys.ToArray());
 				builder.SignPSBT(psbt);
 
-				var isPayjoin = false;
-
 				// Try to pay using payjoin
 				if (payjoinClient is not null)
 				{
 					psbt = TryNegotiatePayjoin(payjoinClient, builder, psbt, changeHdPubKey);
-					isPayjoin = true;
 					psbt.AddKeyPaths(KeyManager);
 					psbt.AddPrevTxs(TransactionStore);
 				}
 
 				psbt.Finalize();
 				tx = psbt.ExtractTransaction();
 
-				var checkResults = builder.Check(tx).ToList();
-				if (!psbt.TryGetEstimatedFeeRate(out var actualFeeRate))
+				if (!psbt.TryGetEstimatedFeeRate(out _))
 				{
 					throw new InvalidOperationException("Impossible to get the fee rate of the PSBT, this should never happen.");
 				}
```

Comment:
You can remove this, too. This sanity check was only added so we can have the sanity check below, but without the thing below it's just noise.

Reply:
Done.